src/app/shared/components/product/components/product-item/product-item.component.css

```diff
@@ -0,0 +1,20 @@
+/* stylelint-disable selector-pseudo-element-no-unknown */
+:host ::ng-deep [slot='id'] {
```

Comment:
Never use `::ng-deep`. It`s considered an antipattern as it leads to css leaking.

Reply:
Also forget about targeting by slot attribute. You are not actually using a slot in angular.
Find the appropiate html element by schemantics (if you need to apply special styling, consider a tailwind style or a new element if there is enogh responsibility